src/schemathesis/runner/impl/core.py

```diff
@@ -114,7 +115,10 @@ def run_test(
         test, "_hypothesis_internal_use_generated_seed", None
     )
     results.append(result)
-    yield events.AfterExecution.from_result(result=result, status=status, hypothesis_output=hypothesis_output)
+    test_elapsed_time = time.monotonic() - test_start_time
```

Comment:
I think that it will be better to measure the test running time without other lines in this function, so only wrapping the test execution line should be ok (before the test and after `except` parts).